tests/.data/README.rst

```diff
@@ -0,0 +1,2 @@
+Resolwe doesn't create `data` and `upload` directories automatically so
```

Comment:
I'll accept this because we probably want an upload and data dirs here even if we add them to resolwe later.